keyboards/gboards/gergoplex/keymaps/colemak-dhm/rules.mk

```
@@ -0,0 +1,17 @@
+#----------------------------------------------------------------------------
+# make gboards/gergoplex:default:flash
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+
+#Debug options
+VERBOSE                = no
+DEBUG_MATRIX_SCAN_RATE = no
+DEBUG_MATRIX           = no
+CONSOLE_ENABLE         = no
+
+#Combos!
+VPATH                  +=  keyboards/gboards/
+
+ifeq ($(strip $(DEBUG_MATRIX)), yes)
+    OPT_DEFS += -DDEBUG_MATRIX
+endif
```

Enable [community layout](https://beta.docs.qmk.fm/developing-qmk/qmk-reference/feature_layouts) support:

```suggestion
endif

LAYOUTS = split_3x5_3
```